packages/core/src/schema/models/evaluationResults.ts

```diff
@@ -26,8 +26,8 @@ export const evaluationResults = latitudeSchema.table(
     providerLogId: bigint('provider_log_id', { mode: 'number' })
       .notNull()
       .references(() => providerLogs.id),
-    resultableType: evaluationResultTypes('resultable_type').notNull(),
-    resultableId: bigint('resultable_id', { mode: 'number' }).notNull(),
```

Comment:
Talked with @csansoon we want to capture errors and also persist the document logs or evaluation results no matter what. This way we document that something happened also we document the duration, what document and with what parameters that error occurred.

Reply:
For that reason we need to make from now on results nullable